vllm_ascend/ascend_config.py

```diff
@@ -45,6 +45,11 @@ def __init__(self, vllm_config):
             ascend_scheduler_config)
 
         self.expert_map_path = additional_config.get("expert_map_path", None)
+        # Eplb config
```

Comment:
Add more note here:

`# TODO: once xxx pr is merged in vLLM. Remove this config`